015-kafka-topic-encryption.md

```diff
@@ -97,38 +93,37 @@ Envoy however is but one viable approach, but certainly not the only means, to e
 in a proxy. 
 
 
-
 ## Affected/not affected projects
 
-Call out the projects in the Strimzi organization that are/are not affected by this proposal. 
-
+This proposal does not conflict or overlap with specific Strimzi projects but rather
+provides a complementary, new functionality which ideally will become part of the
+Strimzi Kafka distribution.
 
 ## Compatibility
 
-Call out any future or backwards compatibility considerations this proposal has accounted for.
+As this is a new capability, there are no backwards compatibility issues relating to this proposal.
```

Comment:
I'd add something like "The system will intercept ProduceRequests and FetchResponses and therefore must make sure that any future changes to the Kafka API in these specific message types are accounted for."

Reply:
Great point. I just pushed a new version and check that I captured your suggestion correctly.